corehq/apps/users/forms.py

```diff
@@ -794,13 +799,27 @@ class CommtrackUserForm(forms.Form):
     )
 
     def __init__(self, *args, **kwargs):
+        from corehq.apps.locations.views import base_queryset_for_domain_and_user, get_locations_paylod_from_ids
+
         self.domain = None
+        populate_with = None
         if 'domain' in kwargs:
             self.domain = kwargs['domain']
             del kwargs['domain']
+
+        if 'user' in kwargs:
```

Comment:
Should this just be a required argument?  That might simplify some things, and I can't think of an instance offhand where we wouldn't have `user` available when instantiating the form.